### PR TITLE
Get rid of expects(big number)

### PR DESCRIPTION
**Pull request by AD7six**
And instead, assert the text looks like the output.

Depends upon https://github.com/cakephp/cakephp/pull/6694

### CODE REVIEW

**Comment by markstory** [comment]
Much nicer :clap:

**Comment by AD7six** [comment]
["Oh"](https://github.com/cakephp/bake/blob/master/.travis.yml#L30)

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": ">=5.4",
-        "cakephp/cakephp": "~3.0"
+        "cakephp/cakephp": "dev-master"
```
Comment (josegonzalez) [inline composer.json]: Why this change? Just curious.

Comment (markstory) [inline composer.json]: I think this was a mistake. I'll revert it.

Comment (markstory) [inline composer.json]: Hmm without relying on dev-master the tests fail because the stub console output is not yet in a packaged release.

Comment (josegonzalez) [inline composer.json]: Well sounds like this depends on 3.1 then?

Comment (josegonzalez) [inline composer.json]: Or at least can't be 3.x.

Comment (lorenzo) [inline composer.json]: I think setting this to dev-master is a mistake, can we revert it?

Comment (AD7six) [inline composer.json]: It is a temporary change so that the tests pass on travis - there is no release with the stub console output yet.

Comment (markstory) [inline composer.json]: I'll probably do 3.0.7 in a week or so. We'll have to not release bake until after that :smile:

**Comment by jeremyspencer** [comment]
screenshot_2015-10-07-04-21-18

**Comment by jeremyspencer** [comment]
https://plus.google.com/109951829811741231558
